core/opds2_import.py

```diff
@@ -100,11 +114,80 @@ def parse_manifest(self, manifest):
         return result
 
 
-class OPDS2Importer(OPDSImporter):
+class OPDS2ImporterConfiguration(ConfigurationGrouping):
+    """Contains configuration settings of ProQuestOPDS2Importer."""
```

Comment:
Comment seems incorrect. I don't think these settings are scoped to the proquest importer.

Reply:
Actually saw this during the merge and forgot about it later. Fixed.